src/plugin.js

```diff
@@ -41,6 +55,19 @@
     }
   }
 
+  Object.keys(backfillClasses).forEach(function(className) {
+    backfillClasses[className].forEach(function(name) {
+      var Component = videojs.getComponent(name);
+      var createEl = Component.prototype.createEl;
+
+      Component.prototype.createEl = function() {
```

Comment:
There's no guarantee that all components are in place on player init. For instance, what if they get generated by a plugin at some later point? This approach seems way more foolproof.